boards/lilygo/ttgo_t7v1_5/ttgo_t7v1_5_esp32_appcpu.dts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32/esp32_appcpu.dtsi>
+
```

Comment:
Pease, add 
`#include <espressif/partitions_0x1000_amp.dtsi>` 
and remove `flash0` entry below.

Reply:
done